src/src/auth/dto/username-otp.dto.ts

```diff
@@ -0,0 +1,14 @@
+import {
```

Comment:
lets rename this file as `get-mobile-by-username-send-otp.dto.ts`

class name as
`GetMobileByUsernameSendOtpDTO`